subjects/devops/better-cat/README.md

```diff
@@ -0,0 +1,42 @@
+## better-cat
+
+### Instructions
+
+Create a script `better-cat.sh` that will behave similarly to the `cat` command, but with additional functionality.
+
+The script should accept any number of filenames as arguments, and it should print the content of each file to the standard output. If no arguments are provided, the script should print the content of all files in the current directory.
```

Comment:
It is clear from the example provided, but it might be helpful to know that the script should `... print the content of each file ...` **preceded by the line number of the file** (or something similar) `... to the standard output`

Reply:
Thanks, you are right, this is specified in the jira ticket and it can be seen in the usage examples but is never being asked directly. I'll add it :+1: